neo/Prompt/Commands/LoadSmartContract.py

```diff
@@ -224,16 +224,10 @@ def ImportMultiSigContractAddr(wallet, args):
     m = get_arg(args, 1)
     publicKeys = args[2:]
 
-    if publicKeys[1]:
-        pubkey_script_hash = Crypto.ToScriptHash(pubkey, unhex=True)
-
-        verification_contract = Contract.CreateMultiSigContract(pubkey_script_hash, int(m), publicKeys)
-
-        address = verification_contract.Address
-
-        wallet.AddContract(verification_contract)
-
-        print("Added multi-sig contract address %s to wallet" % address)
-        return address
+    pubkey_script_hash = Crypto.ToScriptHash(pubkey, unhex=True)
```

Comment:
What would also improve this code is:
- catching the possible exception thrown by `Contract.CreateMultiSigcontract` when `m` != `len(publicKeys)`

Reply:
Hehe, this indeed solves it. I should have been more explicit because if you look in the code you'll see it does more than just that
https://github.com/CityOfZion/neo-python/blob/e002fa580388aedb6b83ef8fd8a4a2867b858813/neo/SmartContract/Contract.py#L91-L98

In stead of replicating that logic I'd rather see a: try/except around
`verification_contract = Contract.CreateMultiSigContract(pubkey_script_hash, int(m), publicKeys)`

While you're at it, can you please change the `raise Exception`'s to `raise ValueError`'s ? Thanks!

Reply:
I tried for quite awhile, but I couldn't figure out how to mock the `len(publicKeys) > 1024` test. I don't think it is worth creating a list with a length that is actually 1025. Let me know if you have any more recommendations.